rust/rustfmt.toml

```diff
@@ -1,3 +1,4 @@
+# requires unstable rustfmt until the options are stabilized
```

Comment:
Have you tried the option `unstable_features=true`? [unstable_features](https://github.com/rust-lang/rustfmt/blob/master/Configurations.md#unstable_features) Maybe it allows us to run fmt via `cargo fmt` instead of `cargo +nightly fmt`.

Reply:
i've tried that, but it doesn't work. i found an open issue on the `rustfmt` github which says that this option itself is not stable and therefore doesn't have an effect for stable toolchains.

Reply:
Alright 🙈